assets/js/components/notifications/AdSenseAlerts.js

```diff
@@ -39,60 +39,56 @@ function AdSenseAlerts() {
 	const accountID = useSelect( ( select ) =>
 		select( MODULES_ADSENSE ).getAccountID()
 	);
+
+	const [ registeredNotifications, setRegisteredNotifications ] = useState(
+		[]
+	);
+	const { registerNotification } = useDispatch( CORE_NOTIFICATIONS );
+
 	const notifications = useSelect( ( select ) =>
 		select( MODULES_ADSENSE ).getNotifications()
 	);
 
-	if (
-		! adSenseModuleConnected ||
-		! accountID ||
-		notifications === undefined
-	) {
-		return null;
-	}
+	useEffect( () => {
+		// If AdSense is not connected, or the account ID is not set,
+		// there's nothing to do here.
+		if ( ! adSenseModuleConnected || ! accountID ) {
```

Comment:
I don't think it's possible for it to be connected without an account ID but I see this is what it did before 🤔